src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -57,7 +57,7 @@ class TiStrategy(context: SQLContext) extends Strategy with Logging {
     selectRequest.setTableInfo(table)
     // In case count(*) and nothing pushed, pick the first column
     if (selectRequest.getFields.size == 0) {
-      selectRequest.addField(TiColumnRef.create(table.getColumns.get(0).getName))
+      selectRequest.addRequiredColumn(TiColumnRef.create(table.getColumns.get(0).getName))
```

Comment:
this change is not merged to master yet. its not a inconsistency

Reply:
Yeah. You are right. Just wait `index_read` merging into master. If there are still some inconsistency, it should be address. Otherwise, we can just close this PR.